pyproject.toml

```
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "2.0.0-beta.2"
+version = "2.0.0rc1"
```

Do we need the `-` between? 
```suggestion
version = "2.0.0-rc1"
```